### ISSUE TITLE
Custom Avatars gone?

### ISSUE THREAD

**Issue by Fohdeesha** [jitsi/jitsi-meet#935]
We used to be able to set our own custom avatars by providing a URL to a picture. However I just ran an apt-get update/upgrade for the first time in about a month today and now that option in jitsi is gone. it seems there were some changes 25 days ago to jitsi-meet/modules/UI/avatar/ and that seemingly took away custom avatar options.

Anything I can do?

**Comment by damencho** [comment]
This wasn't supposed to be available for setting through the UI. I'm not sure whether there is an option to do it currently without UI, I need to check. 
Please use the mailinglist when having a question or a problem, before opening an issue.

**Comment by damencho** [comment]
I checked and it is currently not possible to use an url as an avatar. It was left in the code only for those that use the token and the ring overlay, so external systems can set the avatar. 
There were also security concerns about the URL and that is why it was removed, as user can insert a bad url for the other participants and also one can collect stuff like ip addresses, browser and OS versions from participants in the conference, but if the user adding the URL is authenticated using the token this is fine, cause there is a track of what and who was doing what.
